en/tutorials-and-examples/blog/part-two.rst

```diff
@@ -106,10 +106,11 @@ access that at www.example.com/articles/foobar.
     readable, understandable action names. You can map URLs to your code using
     :doc:`/development/routing` covered later on.
 
-The single instruction in the action uses ``set()`` to pass data
-from the controller to the view (which we'll create next). The line
-sets the view variable called 'articles' equal to the return value of
-the ``find('all')`` method of the ``ArticlesTable`` object.
+The single instruction in the action uses ``set()`` to pass resultset
+from the controller to the view (which we'll create next). The ``find()`` method 
+of the ``ArticlesTable`` object returns an instance of ``Cake\\ORM\\Query`` and
+calling the query's ``all()`` instance returns as instance of ``Cake\Datasource\\ResultSetInterface``
```

Comment:
```suggestion
calling ``all()`` returns as instance of ``Cake\Datasource\\ResultSetInterface``
```
ResultSetInteface is an awkward type to introduce to users. If you look up the API, there's no documentation on it, and it's not mentioned anywhere else. We need to link to something explaining result iteration.

Reply:
I agree, it's not ideal. I avoided stating that a ResultSet instance is returned as it can also be a ResultSetDecorator instance.

Reply:
The common interface is CollectionInterface so maybe we should promote that instead of some implementation details.